doc/mediasdk-man.md

```diff
@@ -4457,32 +4452,10 @@ This structure is available since SDK API 1.1.
 **Definition**
 
 ```C
-/* MCTFTemporalMode */
-enum {
-    MFX_MCTF_TEMPORAL_MODE_UNKNOWN  = 0,
-    MFX_MCTF_TEMPORAL_MODE_SPATIAL  = 1,
-    MFX_MCTF_TEMPORAL_MODE_1REF     = 2,
-    MFX_MCTF_TEMPORAL_MODE_2REF     = 3,
-    MFX_MCTF_TEMPORAL_MODE_4REF     = 4
-};
-
-/* MVPrecision */
-enum {
-    MFX_MVPRECISION_UNKNOWN    = 0,
-    MFX_MVPRECISION_INTEGER    = (1 << 0),
-    MFX_MVPRECISION_HALFPEL    = (1 << 1),
-    MFX_MVPRECISION_QUARTERPEL = (1 << 2)
-};
-
 typedef struct {
     mfxExtBuffer Header;
     mfxU16       FilterStrength;
-    mfxU16       Overlap;            /* tri-state option */
```

Comment:
hm. Why we are dropping these APIs? Aren't they in 1.33? I don't quite get the idea...

Reply:
It is different part.